### PR TITLE
[#13][UI] As a user, I can response to single-select survey type

### LINKED ISSUE #13: [UI] As a user, I can response to single-select survey type
## Why
The application supports various types of survey items. After the user fills in all required survey items, the survey can be submitted and mark the survey as completed.

## Acceptance Criteria
- Display answer section for `choice` survey type
image

- Store the user's response for survey submission

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #13 

## What happened 👀

- Add a single select answer type
- Add simple-bar dependency to ensure the scrollbar will be the same across the browsers
 
## Proof Of Work 📹

The answer can be submitted

https://user-images.githubusercontent.com/14077479/177912216-e7bf75fa-2696-4e16-9988-abbe3475ef34.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "sass": "1.45.1",
+    "simplebar-react": "2.4.1",
```
Comment (hanam1ni) [inline package.json:34]: This dependency is to display the same scrollbar for every browser

File: src/components/SurveyAnswer/ChoiceAnswer.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  const answerStateClass = (answerId: string) => {
+    const isActiveAnswer = currentResponse?.answers[0].id === answerId;
+
+    if (isActiveAnswer) {
+      return 'border-y border-gray-300 text-white font-bold';
+    }
+
+    return 'text-gray-400';
+  };
```
Comment (malparty) [inline src/components/SurveyAnswer/ChoiceAnswer.tsx:24]: Should we add some tests to cover the answerStateClass as well as the onAnswerSelect logic? 🤔

**Review by malparty — CHANGES_REQUESTED** [review]
Minor suggestion :)

File: src/components/SurveyAnswer/ChoiceAnswer.tsx (hunk 2 above)
Comment (hanam1ni) [reply src/components/SurveyAnswer/ChoiceAnswer.tsx:24]: The onAnswerSelect logic is already covered in `click on answer` test case. So I added the test to cover the active and inactive states here [6f9cb5f](https://github.com/hanam1ni/react-surveyor/pull/82/commits/6f9cb5f85fb04d9561ba950636c72bc30ad34d82)

Comment (malparty) [reply src/components/SurveyAnswer/ChoiceAnswer.tsx:24]: Oh yes, true! Thanks!

**Review by malparty — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]

**Pull request merged**